tests/unit/lms/services/group_info_test.py

```diff
@@ -39,23 +43,25 @@ def test_it_updates_an_existing_GroupInfo_if_one_already_exists(
             factories.HGroup(
                 authority_provided_id=self.AUTHORITY, type="course_group",
             ),
-            consumer_key=self.CONSUMER_KEY,
+            consumer_key=application_instance.consumer_key,
```

Comment:
This is doing the same work (removing a magical string), but better because it also tells you where it came from.